.circleci/config.yml

```diff
@@ -308,7 +309,7 @@ workflows:
             - nodejs-install
             - team-lumos
             - snyk-bot-slack
-          node_version: "12"
+          node_version: "14"
```

Comment:
will we do an ugrade to the current LTS version is stages?

Reply:
that would be 18 or 20

Reply:
Eventually I hope 🤞 
I was upgrading it to 14 as the latest `jest` minimum node is 14.